scripts/build-kernel.sh

```diff
@@ -44,4 +44,8 @@ done
 touch .scmversion
 cp $ROOTDIR/aero-kernel/patches-4.4/defconfig .config
 yes '' | make oldconfig
-make deb-pkg -j5
+rm -rf debian/
+cp -rf $ROOTDIR/aero-kernel/debian ./debian
+dpkg-buildpackage -S
+mkdir -p $ROOTDIR/ppa && cd $ROOTDIR/build/ && cp linux*.changes linux*.dsc linux*.tar.gz $ROOTDIR/ppa
```

Comment:
I think we decided to have kernel-ppa folder separate as contents of this directory will be overwrite after running make deb-pkg

Reply:
Updated.